book/01-introduction/sections/about-version-control.asc

```diff
@@ -1,23 +1,23 @@
-=== About Version Control
+=== Versiyon Kontrol Sistemleri
 
 (((version control)))
-What is ``version control'', and why should you care?
-Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later.
-For the examples in this book, you will use software source code as the files being version controlled, though in reality you can do this with nearly any type of file on a computer.
+"Versiyon kontrol" nedir ve neden önemsenmelidir?
+Versiyon kontrol sistemi; dosya ve dosya grupları üzerinde yapılan değişiklikleri versiyonlar şeklinde kayıt edip istenilen zamanda ilgili versiyonlara erişilebilmesini sağlayan sistemdir.
+Örneğin bu kitapta, versiyon kontrol sistemi için yazılım kodları kullanılacaktır fakat versiyon kontrol sistemi bilgisayarda ki herhangi bir dosya türü için de kullanılabilir.  
 
-If you are a graphic or web designer and want to keep every version of an image or layout (which you would most certainly want to), a Version Control System (VCS) is a very wise thing to use.
-It allows you to revert selected files back to a previous state, revert the entire project back to a previous state, compare changes over time, see who last modified something that might be causing a problem, who introduced an issue and when, and more.
-Using a VCS also generally means that if you screw things up or lose files, you can easily recover.
-In addition, you get all this for very little overhead.
+Eğer grafik yada web tasarımcı iseniz ve çalıştığınız görüntü yada tasarımların her bir değişikliklerini tutmak istiyorsanız (ki bu gerçekten istebilecek bir şeydir), Versiyon Kontrol Sistemi (VCS) mantıklı bir seçim olacaktır.
```

Comment:
> grafik yada web tasarımcı 
> görüntü yada tasarımların

`ya da` ayri.

--

> mantıklı

`wise`, mantıklıdan ziyade akillica gibi bir anlama geliyor.

Reply:
wise için yeni çeviri, 
ya da ayrımı
> Eğer grafik ya da web tasarımcı iseniz ve çalıştığınız görüntü ya da tasarımların her bir değişikliklerini tutmak istiyorsanız (ki bu gerçekten istebilecek bir şeydir), Versiyon Kontrol Sistemi (VKS) akıllıca bir seçim olacaktır.